pkg/kubelet/certificate/kubelet.go

```diff
@@ -63,6 +63,29 @@ func NewKubeletServerCertificateManager(kubeClient clientset.Interface, kubeCfg
 	)
 	legacyregistry.MustRegister(certificateExpiration)
 
+	certificateRotationAge := compbasemetrics.NewHistogram(
+		&compbasemetrics.HistogramOpts{
+			Namespace: metrics.KubeletSubsystem,
+			Subsystem: "certificate_manager",
```

Comment:
Why are you puting KubeletSubystem in namespace and not subsystem?

Reply:
The strings _Namespace_, _Subsystem_, and _Name_ are just concatenated.

[See the function.](https://github.com/prometheus/client_golang/blob/master/prometheus/metric.go#L102)

So it doesn't actually matter whether its a namespace or system, all that matters is that they are in the same order. I changed it to be less confusing but functionally the same.